apps/experiments/migrations/0121_remove_experiment_assistant.py

```
@@ -0,0 +1,17 @@
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('experiments', '0120_backfill_session_fields'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='experiment',
+            name='assistant',
+        ),
+    ]
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)